Dockerfile

```diff
@@ -40,7 +40,9 @@ RUN apk add --no-cache \
       protoc \
       protobuf-dev
 WORKDIR /src
-ENV CGO_ENABLED=0
+# Switching CGO to true as there is a weird issue with the Golang DNS resolver and IPV6
+#https://github.com/golang/go/issues/52839
+ENV CGO_ENABLED=1
```

Comment:
Setting the env var here will not be taken into account as Makefile already set it: https://github.com/docker/compose/blob/32ae036fd013566cc45ceef35f5f5b7cb29725bc/Makefile#L51